data/README.md

```diff
@@ -0,0 +1,107 @@
+# Dataset Documentation
+
```

Comment:
You're missing some important parts of the readme, but that's ok for this stage as long as we're planning to add them. You may not be able to fill all of the other parts out — would be good to flag what info you'll need from me or need me to dig up.

You can see the [structure I used for S&co or](https://dataspace.princeton.edu/bitstream/88435/dsp01fx719q532/1/SCoData_combined_v1.1_2021-01_README.txt) — probably better — get a current dataset readme template from PRDS.  (The S&co one was based on a template they shared from somewhere else, but I think they have a newer one).

Reply:
to be clear — I'm fine with that part of the readme being edited/revised in a google doc and then added back to github when it's settled

Reply:
Matt Chandler says he prefers an intermediate version, and said that he'll share his preferred README structure. This draft was easy to cook up with the script that you wrote!